packages/react-server-examples/bike-share/api/index.js

```diff
@@ -0,0 +1,13 @@
+import { getNetworks, getNetwork } from "./networks";
+
+function setJsonContentType(req, res, next) {
+	res.type('json');
+	next();
+}
+
+export default function registerCustomMiddlewares(server, registerRenderMiddleware) {
+	server.use('/api', setJsonContentType);
```

Comment:
Was going to suggest putting a link here to the section on using custom express middleware, but then realized that you can't link to the headers. Filed #862.